core/src/cats/dml/0037_create_temp_accurate_jobids

```diff
@@ -14,7 +14,7 @@ WHERE ClientId = %s
   AND Level = 'F'
   AND JobStatus IN ('T',
                     'W')
-  AND Type = 'B'
```

Comment:
This might need to be changed to `AND Type = %c` if we decide to keep the change.  Even though no tests did cause errors, this might lead to unintended consequences when you call the function and `jr->JobLevel = Full`.  Ill leave it like this for now until we decide whether to revert the change to accurate or not.

Reply:
Seems like I had already done so.  I should have checked the source first before making the comment!

Reply:
changes kepts